cep47/Cargo.toml

```diff
@@ -33,5 +31,4 @@ no_burn_many = []
 no_burn_one = []
 no_transfer_token = []
 no_transfer_many_tokens = []
-no_transfer_all_tokens = []
```

Comment:
Can you explain to me the approach here? are we selectively "turning off" certain endpoints if we compile contract with those `no_` features?

Reply:
This is the way to use cep47 as lib. if you turn on that feature then the library doesn't generate the no_mangle function, so you can define it in your contract. We have found no other way of doing that.